test/studies/bale/indexgather/ig.chpl

```diff
@@ -0,0 +1,70 @@
+use CyclicDist;
+use BlockDist;
+use Random;
+use Time;
+
+config const printStats = true,
+             printArrays = false,
+             verify = true;
+
+config const useRandomSeed = true,
+             seed = if useRandomSeed then SeedGenerator.oddCurrentTime else 314159265;
+
+
+config const numTasksPerLocale = here.maxTaskPar;
```

Comment:
Should this also be passed to the distributions?

Reply:
Hmm.. that's a good point. How did I think this was doing anything. I'm just going to make it a const and set --dataParTasksPerLocale=2 in the execopts